java.lang.NoClassDefFoundError: okhttp3/OkHttpClient$Builder

Machine : Linux 4.19.0-9-amd64 #1 SMP Debian 4.19.118-2 (2020-04-29) x86_64 GNU/Linux
Installer : https://github.com/akasolace/HO/releases/download/3.0/HO-3.0.2.2728.deb

Hi, I get this error when clicking on 'League details' (or so, I've got it in Spanish) :

java.lang.NoClassDefFoundError: okhttp3/OkHttpClient$Builder
	at module.series.promotion.HttpDataSubmitter.initializeHttpsClient(HttpDataSubmitter.java:263)
	at module.series.promotion.HttpDataSubmitter.fetchSupportedLeagues(HttpDataSubmitter.java:45)
	at module.series.promotion.LeaguePromotionHandler.isActive(LeaguePromotionHandler.java:50)
	at module.series.promotion.PromotionInfoPanel.lambda$new$1(PromotionInfoPanel.java:43)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.lang.ClassNotFoundException: okhttp3.OkHttpClient$Builder
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 17 more

Thanx

--- a/build.gradle
+++ b/build.gradle
@@ -169,6 +169,9 @@ ospackage {
     from(jar.outputs.files) {
         into 'lib\\ho'
     }
+    from(configurations.runtime) {
+        into 'lib\\ho'
+    }
     from("${projectDir}\\src\\main\\resources\\sprache") {
         into 'lib\\ho\\sprache'
     }
@@ -181,11 +184,6 @@ ospackage {
 
     from("${projectDir}\\src\\main\\resources\\themes") {
         into 'lib\\ho\\themes'
-    }
-
-    from("${projectDir}\\src\\main\\resources") {
-        include "**/*.jar"
-        into "lib\\ho"
     }
 
     from("${projectDir}\\src\\main\\resources") {

--- a/src/main/resources/changelog.md
+++ b/src/main/resources/changelog.md
@@ -10,10 +10,11 @@ If you find a bug, please open an issue on [GitHub](https://github.com/akasolace
 ## Misc
 
 - [FIX] Avoid potential infinite loop at startup. [#584]
+- [FIX] Fix NoClassDefFoundError in Deb package. [#589]
 
 # Changelist HO! 3.0
 
-## Some numbers: 
+## Some numbers:
   - 90 commits
   - 293 files changed (11,027 additions and 6,341 deletions)
   - 42 issues closed
@@ -28,7 +29,7 @@ If you find a bug, please open an issue on [GitHub](https://github.com/akasolace
   - impact of special events on score for both you and your opponent based on latest lineup information
 
   - new match report mocking HT full report
-  
+
   - full control on which game to download (e.g. exclude HTO integrated games)
 
 
@@ -88,7 +89,7 @@ If you find a bug, please open an issue on [GitHub](https://github.com/akasolace
    - [FIX] Index Out Of Bounds Exception at startup on new DBs #448
    - [FIX] fix multiple display issues about player names (composed name, nicknames)  #451
    - [FIX] removed deprecated training block feature  #486
-   
+
 
 ### League
 

--- a/src/main/resources/release_notes.md
+++ b/src/main/resources/release_notes.md
@@ -17,3 +17,4 @@ Changelist HO! 3.1
 ### Misc
 
 - [FIX] Avoid potential infinite loop at startup. [#584]
+- [FIX] Fix NoClassDefFoundError in Deb package. [#589]

